.gitignore

```diff
@@ -1,4 +1,4 @@
 /node_modules
-docs/.vuepress/.cache
-docs/.vuepress/.temp
-docs/.vuepress/dist/*
+./docs/.vuepress/.cache
```

Comment:
```suggestion
/docs/.vuepress/.cache
```

You just need a slash here for the .gitignore. Thats why this PR contains youre entire Cache file